e2e/cypress/support/ui_commands.js

```diff
@@ -421,3 +421,111 @@ Cypress.Commands.add('fileUpload', (targetInput, fileName = 'mattermost-icon.png
         );
     });
 });
+
+/**
+ * Upload a file on target input in binary format -
+ * @param {String} Subject - cypress/fixtures/matterMost.tar.gz
+ * @param {String} filePath - cypress/fixtures/matterMost.tar.gz
+ * @Param {String} fileType - application/gzip
+ */
+Cypress.Commands.add('uploadFile2', {prevSubject: true}, (subject, fileName, fileType) => {
```

Comment:
Please include all these functions directly in the testcase being used. Only functions which are very common and used across multiple tests is good to be added here.

Reply:
Function specific to test are updated in the testcase